tensorbay/opendataset/BDD100K/loader.py

```diff
@@ -405,29 +400,31 @@ def _save_and_get_mask_info(
     original_mask_path: str, mask_path: str, mask_info_path: str, seg_type: str
 ) -> Dict[str, Any]:
     if not os.path.exists(mask_path):
-        mask = np.array(Image.open(original_mask_path))
+        mask = np.array(Image.open(original_mask_path), dtype=np.uint16)
         all_attributes = {}
         if seg_type == "pan":
             all_category_ids = {}
-        for instance_info in set(map(tuple, np.reshape(mask, (-1, 4)))):
-            instance_id = int(instance_info[-1])  # type:ignore[call-overload]
-            attributes = instance_info[1]
+        for category_id, attributes, instance_id_high, instance_id_low in np.unique(
+            np.reshape(mask, (-1, 4)), axis=0
```

Comment:
can we use the `axis` parameter in `np.unique` rather than `np.reshape`?

Reply:
`np.unique` can't be applied on 3 dimensional array, so I need to reshape it first